manifests/master.pp

```diff
@@ -4,9 +4,15 @@
 #
 # Includes all subordinate classes.
 class profile_xcat::master {
-    include ::profile_xcat::master::bmc_smtp
-    include ::profile_xcat::master::firewall
-    include ::profile_xcat::master::nfs
-    include ::profile_xcat::master::root
+  include ::profile_xcat::master::bmc_smtp
+  include ::profile_xcat::master::firewall
+  include ::profile_xcat::master::nfs
+  include ::profile_xcat::master::root
+
+  # tcpwrappers was removed in RHEL8
+  $os_family = $facts['os']['family']
+  $os_rel_major = Integer( $facts['os']['release']['major'] )
+  if $os_family == 'RedHat' and $os_rel_major < 8 {
     include ::profile_xcat::master::tcpwrappers
+  }
```

Comment:
In other modules we have just removed tcpwrappers entirely. Do we really need this?

Reply:
I don't think so ... left it for possible ease of security vetting, so it get's applied on RHEL7 (that still supports tcpwrappers). Seemed an easy route to take since the code was already there. 

I will open an enhacement request (issue) to remove it.

Reply:
Opened #2